README.md

```diff
@@ -3,70 +3,62 @@
 [![Build Status](https://travis-ci.com/PsychoinformaticsLab/neuroscout.svg?token=mytABRBRnBitJJpBpMxh&branch=master)](https://travis-ci.com/PsychoinformaticsLab/neuroscout)
 [![codecov](https://codecov.io/gh/PsychoinformaticsLab/neuroscout/branch/master/graph/badge.svg)](https://codecov.io/gh/PsychoinformaticsLab/neuroscout)
 
-To set up docker, ensure docker and docker-compose are installed.
-First, set up your mounted volumes (such as /datasets), by exporting environment variables:
+This is the repository for the neuroscout server.
 
-    DATASET_DIR=/home/myuser/datasets
-    export DATASET_DIR
+Requirements: Docker and docker-compose.
 
-To set up environment variables for pliers, create a file called .pliersenv, and
-configure your private keys for pliers there.
+## Seting up environment / config variables
+First, set up the main environment variables in `.env` (see: `.env.example`).
+Set `DATASET_DIR`, `KEY_DIR`, and `FILE_DATA` to folders on the host machine.
 
-In addition, configure where file keys (such as for Google) are located:
+Optionally, set up pliers API keys for feature extraction in `.pliersenv` (see: `.pliersenv.example`).
+[More information on pliers API keys](http://tyarkoni.github.io/pliers/installation.html#api-keys)
 
-    KEY_DIR=/home/me/.keys
-    export KEY_DIR
+Next, set up the Flask server's environment variables by modifying `neuroscout/config/example_app.py`
+and saving as `neuroscout/config/app.py`.
 
+Finally, set up the frontend's env variables by modifying `neuroscout/frontend/src/config.ts.example`
```

Comment:
Might want to mention here that they'll need to set up a google sign-in project here:
https://developers.google.com/identity/sign-in/web/sign-in